### PR TITLE
docs: add august contributions showcase

### PR DESCRIPTION
**Pull request by 43081j**

### CODE REVIEW

File: docs/blog/august-contributions-showcase.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+Usage is familiar and simple:
+
+```ts
+import { x } from 'tinyexec'
+
+await x('npm', ['install'])
+```
+
+In many places where we need to launch a command, we don't need a high level abstractions so we can save a lot on performance and footprint by using this instead.
```
Comment (ziebam) [inline docs/blog/august-contributions-showcase.md:42]:
```suggestion
In many places where we need to launch a command, we don't need a high level abstraction so we can save a lot on performance and footprint by using this instead.
```
or
```suggestion
In many places where we need to launch a command, we don't need high level abstractions so we can save a lot on performance and footprint by using this instead.
```

**Review by ziebam — COMMENTED** [review]
Just a minor typo I noticed.

File: docs/blog/august-contributions-showcase.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+# picocolors
+
+[picocolors](https://github.com/alexeyraspopov/picocolors) has been around a while already, but recently published a new version with support for bright variants of colours.
+
+This has allowed many, many projects to migrate to it from other slower and larger libraries.
+
+If you need terminal colors, this is the one to use.
+
+If you're using node 20 or above, and don't need to support older runtimes, also check out [styletext](https://nodejs.org/api/util.html#utilstyletextformat-text-options) which is built in!
```
Comment (bluwy) [inline docs/blog/august-contributions-showcase.md:128]:
```suggestion
If you're using node 20 or above, and don't need to support older runtimes, also check out [styleText](https://nodejs.org/api/util.html#utilstyletextformat-text-options) which is built in!
```

File: docs/blog/august-contributions-showcase.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      content: website
+  - - meta
+    - property: og:title
+      content: August contributions showcase
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/august-contributions-showcase
+  - - meta
+    - property: og:description
+      content: A showcase of e18e-focused contributions from August
```
Comment (benmccann) [inline docs/blog/august-contributions-showcase.md:19]: an og image would help for sharing on twitter, etc.

File: docs/blog/august-contributions-showcase.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+```ts
+import { x } from 'tinyexec'
+
+await x('npm', ['install'])
+```
+
+In many places where we need to launch a command, we don't need a high level abstraction so we can save a lot on performance and footprint by using this instead.
+
+[vitest](https://github.com/vitest-dev/vitest/), [nuxt](https://github.com/nuxt/nuxt) and many others have already moved over.
```
Comment (benmccann) [inline docs/blog/august-contributions-showcase.md:44]:
```suggestion
[vitest](https://github.com/vitest-dev/vitest/), [nuxt](https://github.com/nuxt/nuxt), [astro](https://github.com/withastro/astro), and many others have already moved over.
```

File: docs/blog/august-contributions-showcase.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+- [unocss](https://github.com/unocss/unocss)
+- [pkg-pr-new](https://github.com/stackblitz-labs/pkg.pr.new)
+- [tsup](https://github.com/egoist/tsup)
+- and many, many more
+
+Great work by both [@thecodrr](https://x.com/thecodrr) and [@superchupudev](https://x.com/superchupu) here!
+
+# package-manager-detector
+
+[`package-manager-detector`](https://github.com/antfu-collective/package-manager-detector) is used to detect the local package manager (e.g. `npm`).
```
Comment (benmccann) [inline docs/blog/august-contributions-showcase.md:78]:
```suggestion
[`package-manager-detector`](https://github.com/antfu-collective/package-manager-detector) is used to detect the local package manager (e.g. `npm`, `pnpm`, `yarn`, etc.)
```

File: docs/blog/august-contributions-showcase.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+---
+
+# August contributions
+
+_September 7, 2024_
+
+We've seen another great month of contributions from the community :tada: Here's just a few highlights.
+
+If you want to get involved, remember to come [join the discussion](https://chat.e18e.dev)!
+
```
Comment (benmccann) [inline docs/blog/august-contributions-showcase.md:29]: maybe we could add a heading level to differentiate libraries to be aware of vs packages that are getting cleaned up
```
# Libraries to be aware of

## tinyexec
...

## tinyglobby / fdir
...
...

# Package cleanup efforts

## changesets
...
```

File: docs/blog/august-contributions-showcase.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+[knip](https://github.com/webpro-nl/knip) is a tool for detecting unused files and dependencies in your project. This has been incredibly useful and widely used across the e18e community already.
+
+For example, [mocha](https://github.com/mochajs/mocha/pull/5042) has already adopted this and cleaned up a huge amount of deep dependencies thanks to the efforts of [@voxpelli](https://x.com/voxpelli).
+
+# dotenvx
+
+The maintainers of [dotenvx](https://github.com/dotenvx/dotenvx) have been hard at work with [@superchupudev](https://x.com/superchupu) to massively reduce the install size and depedency tree complexity.
+
+Down from [142 dependencies](https://npmgraph.js.org/?q=@dotenvx/dotenvx@1.0.0) to only [28 dependencies](https://npmgraph.js.org/?q=@dotenvx/dotenvx), this is a great achievement :tada:
+
```
Comment (benmccann) [inline docs/blog/august-contributions-showcase.md:111]: payload cms could be another good one to mention
https://npmgraph.js.org/?q=payload - 579 dependencies
https://npmgraph.js.org/?q=payload@3.0.0-beta.100 - 132 dependencies

the payload team did all the work on that one. Some of its dependencies like bcherny's `json-schema-to-typescript` were also cleaned up

File: docs/blog/august-contributions-showcase.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+Great work by both [@thecodrr](https://x.com/thecodrr) and [@superchupudev](https://x.com/superchupu) here!
+
+# package-manager-detector
+
+[`package-manager-detector`](https://github.com/antfu-collective/package-manager-detector) is used to detect the local package manager (e.g. `npm`).
+
+Much lighter than other alternatives, this package has quickly been adopted by many projects across the ecosystem.
+
+As an example, [@benjaminmccann](https://x.com/benjaminmccann) switched from `preferred-pm` (1MB) to `package-manager-detector` (28KB) in [changesets](https://github.com/changesets/changesets/pull/1446) recently. Great work :pray:
```
Comment (benmccann) [inline docs/blog/august-contributions-showcase.md:82]:
```suggestion
As an example, [@benjaminmccann](https://x.com/benjaminmccann) switched from `preferred-pm` (1MB) to `package-manager-detector` (28KB) in [`changesets`](https://github.com/changesets/changesets/pull/1446) and [`svelte-add`](https://github.com/svelte-add/svelte-add/pull/535) recently. Great work :pray:
```

File: docs/blog/august-contributions-showcase.md (hunk 3 above)
Comment (43081j) [reply docs/blog/august-contributions-showcase.md:19]: For sure. I'm not very good with images so any help would be appreciated 😅

File: docs/blog/august-contributions-showcase.md (hunk 6 above)
Comment (43081j) [reply docs/blog/august-contributions-showcase.md:29]: Good idea. I'll shuffle things around 👍

File: docs/blog/august-contributions-showcase.md (hunk 7 above)
Comment (43081j) [reply docs/blog/august-contributions-showcase.md:111]: Will get it added 👍

**Pull request merged**